<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::MobileAdapterView</class>
 <widget class="QDialog" name="QGBA::MobileAdapterView">
  <property name="windowTitle">
   <string>Mobile Game Boy Adapter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="statusTab">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_statusTab">
       <item>
        <widget class="QGroupBox" name="statusBox">
         <layout class="QVBoxLayout" name="verticalLayout_statusBox">
          <item>
           <widget class="QLabel" name="statusText">
            <property name="text">
             <string>{status}</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="numbersForm" native="true">
         <layout class="QFormLayout" name="formLayout_numbersForm">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_userNumber">
            <property name="text">
             <string>Your number is</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="userNumber">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_peerNumber">
            <property name="text">
             <string>Currently calling</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="peerNumber">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_statusTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="versionWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_versionWidget">
          <item>
           <spacer name="horizontalSpacer_versionWidget">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_versionText">
            <property name="text">
             <string>libmobile version: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="versionText">
            <property name="text">
             <string>{version}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_settingsTab">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_setType">
         <property name="text">
          <string>Adapter type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="setType">
         <item>
          <property name="text">
           <string>Blue (PDC)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow (cdmaOne)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green (PHS-NTT, unreleased)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red (DDI)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Purple (unreleased)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black (unreleased)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pink (unreleased)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grey (unreleased)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="setUnmetered">
         <property name="text">
          <string>Unmetered</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="setDns1"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_setDns1">
         <property name="text">
          <string>Primary DNS:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="setDns2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_setDns2">
         <property name="text">
          <string>Secondary DNS:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="setPort">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1027</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_setPort">
         <property name="text">
          <string>P2P port:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="setRelay"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_setRelay">
         <property name="text">
          <string>Relay server:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="setToken">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_setToken">
         <property name="text">
          <string>Relay token:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="copyToken">
         <property name="text">
          <string>Copy relay token</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
